Makefile

```diff
@@ -1,40 +1,45 @@
+ifdef CI
+	DOCKER_PHP_EXEC :=
+else
+	DOCKER_PHP_EXEC := docker compose run --rm php
+endif
 PARALLELISM := $(shell nproc)
 
 .PHONY: all
-all: install phpcbf phpcs phpstan phpunit infection phpbench
+all: phpcbf phpcs phpstan phpunit infection phpbench
 
-.PHONY: install
-install: vendor/composer/installed.json
```

Comment:
There's no need to remember to run `make install`, just add `vendor/composer/installed.json` as a prerequisit for every other target